airflow/configuration.py

```diff
@@ -1481,77 +1788,24 @@ def initialize_config() -> AirflowConfigParser:
     Called for you automatically as part of the Airflow boot process.
     """
     global FERNET_KEY, AIRFLOW_HOME, WEBSERVER_CONFIG
```

Comment:
```suggestion
    global WEBSERVER_CONFIG
```

These aren't used any longer in this scope.

Reply:
Right.